### PR TITLE
Updating README.md to the latest action version and adding new comment warning about it

### PR DESCRIPTION
**Pull request by yagoazedias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Docker build and push to ECR
-        uses: olxbr/aws-ecr-push-action@v0.3
+        # Check the latest action version and replace it here.
+        uses: olxbr/aws-ecr-push-action@v0.10
```
Comment (marciorgb) [inline README.md:25]: I believe that we can use the `@master` "tag" to ensure the latest one.

Comment (marciorgb) [reply README.md:25]: or `@main` in this case.

Comment (yagoazedias) [reply README.md:25]: I will test it here and return with a modification

Comment (yagoazedias) [reply README.md]: Done @marciorgb

File: README.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Docker build and push to ECR
-        uses: olxbr/aws-ecr-push-action@v0.3
+        # Check the action version that you want and replace it here.
+        uses: olxbr/aws-ecr-push-action@main
```
Comment (rogerfernandes) [inline README.md:25]: The recommendation is use the Major version like `v0`.

```suggestion
        uses: olxbr/aws-ecr-push-action@v0
```

**Comment by rogerfernandes** [comment]
@yagoazedias I updated the README file. So I'll close this PR, Okay?
Tnks ;)

**Pull request closed without merging**